js-exercises/alternating-characters/alternatingCharacters.js

```diff
@@ -0,0 +1,17 @@
+function alternatingCharacters(arr) {
+  let result=[]
+  arr.map((value)=>{
+    let string=value.split("");
+    let past=string[0];
+    let array=string.filter((value,index)=>{
+      if(value!==past && index!=0){
+        past=value;
+        return past;
+      }
+    })
+    result.push(string.length -[string[0],...array].length)
```

Comment:
lines 5 to 12 ...  It's a complex code and can be simplified further
I hope this will help
![image](https://user-images.githubusercontent.com/17058535/76616599-5e29bb80-654a-11ea-8cfe-2419b2af2c7b.png)